server/src/main/kotlin/com/cuhacking/watershed/server.kt

```diff
@@ -36,20 +53,32 @@ class Main(configPath: String) {
     }
 }
 
-fun Application.installFeatures() {
+fun Application.installFeatures(config: Config, jwtManager: JwtManager) {
     install(CallLogging) {
         level = Level.DEBUG
     }
     install(AutoHeadResponse)
     install(ContentNegotiation) {
         json()
     }
+    install(Authentication) {
+        jwt("auth-jwt") {
```

Comment:
We could consider putting this string value as a constant int `JwtManager`'s companion object.